src/main/kotlin/com/github/mitchellolsthoorn/testgenie/settings/TestGenieSettingsConfigurable.kt

```diff
@@ -1,6 +1,7 @@
 package com.github.mitchellolsthoorn.testgenie.settings
 
 import com.intellij.openapi.options.Configurable
+import com.intellij.openapi.ui.Messages
 import javax.swing.JComponent
 
```

Comment:
(Not related to the place of this comment)
Wasn't there supposed to be a button from the tool window that opens the settings? Something like "Advanced Parameters"?

Reply:
This is currently not a top priority (for the current scope of the sprint). Thanks for the reminder though.